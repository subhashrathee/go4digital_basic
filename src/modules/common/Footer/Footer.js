import React from 'react'

//const date object used
const dt = new Date();

/**
* Footer class used by all page components.
* It has a copyright symbol along with the year
*
* Props:      None
*
* State vars: None
*/
class Footer extends React.Component {
  render() {
    return (
      <footer>  
        <p>&copy; {dt.getFullYear()} go4digital. All rights reserved.</p>
      </footer>
    )
  }
}

export default Footer;
