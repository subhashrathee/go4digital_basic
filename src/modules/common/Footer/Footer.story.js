import React from 'react';
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import { BrowserRouter } from 'react-router-dom'
import Footer from './Footer'

storiesOf('Footer', module)
  .add('Default Footer', () => (
      <Footer />
  ));
