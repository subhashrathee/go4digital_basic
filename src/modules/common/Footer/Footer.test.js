import React from 'react';
import Footer from './Footer';
import renderer from 'react-test-renderer';
import { BrowserRouter, Link } from 'react-router-dom'

test('Footer component', () => {
  const component = renderer.create(
    <Footer />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
