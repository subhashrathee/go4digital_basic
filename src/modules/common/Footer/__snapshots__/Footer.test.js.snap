// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component 1`] = `
<footer>
  <p>
    Â© 
    2018
     go4digital. All rights reserved.
  </p>
</footer>
`;
